src/utils.php

```diff
@@ -35,3 +35,37 @@ function create_list_of_cars(array $cars): string | false
 
     return $ret;
 }
+
+function validateDataFields($name, $year_made, $zero_sixty, $price, $brand): string | true
```

Comment:
This no longer returns true.  
  
Also make sure you add type hints to the params